assets/kube-apiserver/featuregate.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: FeatureGate
+metadata:
+  name: cluster
+spec:
+{% if k8s_feature_gates is defined %}
```

Comment:
Please use the toolkit version of `k8s_feature_gates` ROKS data which is `DefaultFeatureGates` and `ExtraFeatureGates`.